README.md

```diff
@@ -1,2 +1,58 @@
 # Learning materials
-This repository contains learning materials, that is relevant for new engineers at Lunar.
+
+This repository contains learning materials, that are relevant for new engineers at Lunar.
+
+## Content
+
+1. Microservices
+2. Cloud Native
+3. Kubernetes and Containers
+4. Golang
+5. CI/CD and GitOps
+6. Observability
+7. Culture
+8. Other things
+
+## Microservices
+
+**Microservices** by James Lewis and Martin Fowler. https://martinfowler.com/articles/microservices.html.
+
+**Single Team Oriented Service Architecture (STOSA)** by Lee Atchison. https://stosa.org.
+
+## Cloud Native
+
+**CNCF Cloud Native Definition** by CNCF Technical Oversight Committee. https://github.com/cncf/toc/blob/main/DEFINITION.md.
+
+## Kubernetes and Containers
+
+**Borg, Omega, and Kubernetes** by Burns, Brendan, et al. https://storage.googleapis.com/pub-tools-public-publication-data/pdf/44843.pdf.
+
+**What is Kubernetes?** by The Kubernetes Authors. https://kubernetes.io/docs/concepts/overview/what-is-kubernetes.
+
+**Containers** by The Kubernetes Authors. https://kubernetes.io/docs/concepts/containers.
+
+## Golang
+
+**A Tour of Go**. https://go.dev/tour/welcome/1.
+
+**Go by Example**. https://gobyexample.com. (Primarily for reference)
+
+## CI/CD and GitOps
+
+**GitOps - Operations by Pull Request** by Kasper Nissen. https://tech.lunar.app/blog/2020-03-10-gitops-operations-by-pull-request.
+
+**Release Manager** by Lunar. https://github.com/lunarway/release-manager.
+
+**Shuttle** by Lunar. https://github.com/lunarway/shuttle.
```

Comment:
I don't really find these learning materials TBH. They are very specific tools (and not greatly documented as such). I think we should iam for more general articles on these matters.

Reply:
Alright 👍

Reply:
https://github.com/lunarway/learning-materials/pull/3/commits/baf64ff5713f7554a3ed6911af4b0e8b3f5d13d9